crates/sui-framework/sources/EpochTimeLock.move

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+module Sui::EpochTimeLock {
+    use Sui::TxContext::{Self, TxContext};
+
+    /// The epoch passed into the creation of a lock has already passed.
+    const EEPOCH_ALREADY_PASSED: u64 = 0;
+    /// Attempt is made to unlock a lock that cannot be unlocked yet.
+    const EEPOCH_STILL_LOCKED: u64 = 1;
+
+    /// Holder of an epoch number that can only be discarded after the
+    /// epoch has passed.
```

Comment:
The actual implementation seems to be that you could unlock an object at this epoch instead of after the epoch has passed?

Reply:
Good catch. Will update the comment